readthedocs/doc_builder/config.py

```diff
@@ -172,10 +59,27 @@ def load_yaml_config(version):
         # This is a subclass of ConfigError, so has to come first
         raise
     except ConfigError:
+        # TODO: this shouldn't be hardcoded here
+        env_config.update({
+            'output_base': '',
+            'type': 'sphinx',
+            'name': version.slug,
+        })
```

Comment:
Is there anything you should do here before this is merged?

Reply:
I think this would be fixed if we delete this options, they aren't used anyway.

Reply:
So, having hard coded this options here isn't a problem right now, but isn't correct either p: